lib/AI-Agent.xml

```diff
@@ -54,5 +54,15 @@
             <Method name="getWizardPageCollection"/>
             <Method name="getCaseTabCollection"/>
         </Class>
+        <Class name="uk.gov.hmcts.ccd.data.draft.DefaultDraftGateway">
```

Comment:
Dwayne I've noticed we miss a very important class in this file: HttpUIDefinitionGateway
Can we add that please? I'm not sure whether that has already been automatically instrumented since it's http or we must add it explicitly in this file